packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -100,91 +101,84 @@ module.exports = ({
   entryPoint,
   sourceFolders,
   toggleFlags = defaultToggleFlags,
-}) => ({
-  // Don't attempt to continue if there are any errors.
-  bail: true,
+}) => {
+  // NOTE: Without properly defauling passing one flag (e.g. `enabledVendorOptimizations`)
+  // would disable other flags defaulted to `true` (e.g. `enableExtractCss`).
+  const defaultedToggleFlags = {
```

Comment:
Nitpick: `default` and `defaulted` are very similar and it's easier to mistype them. I got myself confused for a couple of seconds. For example, in `test-utils` we name those variables `mergedXxx`. So maybe `mergedToggleFlags`?

Reply:
Sure. I don't mind. It's just how I name them generally so I am used to the naming. Will change.

Reply:
Naming is hard 😇

Reply:
It's sometimes the only fun thing left in programming 😄